llvm/test/lab2/safronov_mikhail/instrumented_test_functions.ll

```diff
@@ -0,0 +1,95 @@
+; RUN: opt -load-pass-plugin %llvmshlibdir/SafronovInstrumentFunctions%pluginext -passes=instr_func -S %s | FileCheck %s
+
+define dso_local void @instrument_start() {
+entry:
+  ret void
+}
+
+define dso_local void @instrument_end() {
+entry:
+  ret void
+}
```

Comment:
Add a test case where these functions do not exist in .ll file

Reply:
struct where these functions do not exist in .ll file

Reply:
There is required are separate test file without definitions of `instrument_start`/`instrument_end` functions to check that their declarations are created in IR on insert of the calls.